app/models/provider.rb

```diff
@@ -74,6 +74,10 @@ class Provider < ApplicationRecord
 
   scope :in_order, -> { order(:provider_name) }
 
+  scope :by_recruitment_cycle, ->(recruitment_year) {
+    joins(:recruitment_cycle).merge(RecruitmentCycle.where(year: recruitment_year))
```

Comment:
is there a need for a join?

given that a provider belongs to a recruitment year?

Reply:
Done 👍